test/unit/org/apache/cassandra/index/sai/disk/vector/VectorCompressionTest.java

```diff
@@ -111,7 +114,9 @@ private void testOne(VectorSourceModel model, int originalDimension, VectorCompr
 
     private void testOne(int rows, VectorSourceModel model, int originalDimension, VectorCompression expectedCompression) throws IOException
     {
-        createTable("CREATE TABLE %s " + String.format("(pk int, v vector<float, %d>, PRIMARY KEY(pk))", originalDimension));
+        createTable(String.format("CREATE TABLE %%s (pk int, v vector<float, %d>, PRIMARY KEY(pk)) " +
+                                  "WITH compaction = {'class': 'UnifiedCompactionStrategy', 'num_shards': 1, 'enabled': false}",
```

Comment:
Just to confirm my understanding: We will add here straight the number of shards in the test to fix the current problem/flakiness, and we will add in `@BeforeClass` the new config for SAI tests in a separate ticket where we will cover all vector tests?